pom.xml

```diff
@@ -467,6 +474,14 @@ under the License.
     </build>
 
     <profiles>
+        <profile>
+            <id>flink-1.14</id>
```

Comment:
Why we need this  @ruanhang1993 , Could we build a connector base on Flink 1.15 and then run it on Flink 1.14 cluster？ I think this is what we want.

Reply:
I will try to remove 1.14 profile.